tests/test_env.py

```diff
@@ -1,20 +1,90 @@
 from __future__ import annotations
 
-from pathlib import Path
+from typing import Dict
 
 import pytest
 
 
-@pytest.fixture()
-def example(testdir: pytest.Testdir) -> pytest.Testdir:
-    src = Path(__file__).parent / "example.py"
-    dest = Path(str(testdir.tmpdir / "test_example.py"))
-    dest.symlink_to(src)
-    return testdir
+@pytest.mark.parametrize("name,existing_env_vars,ini_contents,expected_env_vars", [
+    pytest.param(
+        "test_adds_environment_variable",
```

Comment:
This name can be more meaningful and free text now 🙌

Reply:
Also this should be the id param and not an arg.

Reply:
Both suggestions were applied. Thanks!